app/src/main/res/layout/meal_item_view.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="@dimen/x_large_300"
+    android:layout_height="wrap_content"
+    android:clipToPadding="false">
+
+    <androidx.cardview.widget.CardView
```

Comment:
This CardView should wrap all views included to it - not a hardcoded height and placing views `inside` of it. It can be used as a normal layout: <CardView> .... </CardView> and use a ConstraintLayout inside to create your UI as needed.